recipes/transformer-engine/transformer-engine-env.sh

```diff
@@ -0,0 +1 @@
+export CUDA_HOME=$CONDA_PREFIX
```

Comment:
Would check if `CUDA_HOME` is set and if so keep a copy

```suggestion
if [[ ! -z "${CUDA_HOME +x}" ]]
then
    export CUDA_HOME_BACKUP="${CUDA_HOME}"
fi
export CUDA_HOME="${CONDA_PREFIX}"
```

Then would add a separate deactivate script to restore the original value

```
if [[ ! -z "${CUDA_HOME_BACKUP+x}" ]]
then
  export CUDA_HOME="${CUDA_HOME_BACKUP}"
  unset CUDA_HOME_BACKUP
fi
```

Reply:
sure, done